eventmesh-connector-plugin/eventmesh-connector-pulsar/src/main/java/org/apache/eventmesh/connector/pulsar/config/ClientConfiguration.java

```diff
@@ -25,7 +25,7 @@
 
 @Getter
 @Setter
-@Config(prefix = "eventMesh.server.pulsar", path = "classPath://pulsar-client.properties")
+@Config(prefix = "eventMesh.server.pulsar")
```

Comment:
why you remove this config: path = "classPath://pulsar-client.properties"

Reply:
OK, I will fix it next time I submit